dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -242,20 +242,16 @@ private SQLException classNotFound(Object interfaceNames, Set<String> packagesSe
         if (trace && tc.isEntryEnabled())
             Tr.entry(tc, "create", className, classloader, PropertyService.hidePasswords(props));
 
-        //TODO remove this gate before GA so this also applies to data sources configured using DataSourceDefinition
-        String vendorPropertiesPID = props instanceof PropertyService ? ((PropertyService) props).getFactoryPID() : PropertyService.FACTORY_PID;
-        if ("com.ibm.ws.jdbc.dataSource.properties.oracle.ucp".equals(vendorPropertiesPID)) {
```

Comment:
again, wondering here why we are removing the properties PID type check

Reply:
same as above